content/news/2021-01-08-mixxx-to-the-first-time-contributor.md

```diff
@@ -0,0 +1,110 @@
+title: Mixxx to the first time contributor
+authors: Aanyu Deborah Oduman
+tags: personal, outreachy, internship
+date: 2021-01-08 17:38:00
+summary: Before the Outreachy internship program, I had never contributed to open-source software. I didn’t know what the term "open source" meant or how different it was from other software. Looking back, it's hard to believe how clueless I was four months ago and then compare to how far I've come. I think it is safe to say that I have learned a lot about different open source software, Mixxx in particular, and the different ways that I can contribute. I feel very fortunate to have been offered the Outreachy internship, and now I will share everything that I know about contributing to the Mixxx open source projects - from the perspective of a first time contributor :)
```

Comment:
Is this indended to be part of the summary, but not the actual blogpost?

Reply:
It's a summary that I extracted from the blogpost. Like an excerpt. 
Does it come off repetitive in the actual blogpost ?

Reply:
No, it's just that the blogpost is missing a proper introduction (the first few sentences don't really give an impression what this  post if about).

FYI you don't have to write a summary if there are no major layout elements (embedded images, headlines, etc) at the beginning of your post content.

Reply:
Okay. I think i will just remove it.